### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by blktheta**
Updated README.md with subtitle and some text.

### CODE REVIEW

**Comment by Ugits** [comment]
Looks good !
I will add it

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # Demo
 
 Some dersciption!
+
```
Comment (Ugits) [inline README.md:4]: what does ## do?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+## Forked
+
+Hello Ugit, written from a forked repo.
```
Comment (Ugits) [inline README.md:7]: nice nice

**Review by Ugits — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (blktheta) [reply README.md:4]: Adds a H2 title to the readme,

**Review by Ugits — APPROVED** [review]

**Review by Ugits — APPROVED** [review]

**Pull request merged**